src/southbridge/amd/pi/hudson/bootblock.c

```diff
@@ -6,49 +6,10 @@
 #include <device/pci_ops.h>
 #include <southbridge/amd/pi/hudson/hudson.h>
 
-/*
- * Enable 4MB (LPC) ROM access at 0xFFC00000 - 0xFFFFFFFF.
- *
- * Hardware should enable LPC ROM by pin straps. This function does not
- * handle the theoretically possible PCI ROM, FWH, or SPI ROM configurations.
- *
- * The HUDSON power-on default is to map 512K ROM space.
- *
- */
-static void hudson_enable_rom(void)
-{
-	u8 reg8;
-	const pci_devfn_t dev = PCI_DEV(0, 0x14, 3);
-
-	/* Decode variable LPC ROM address ranges 1 and 2. */
-	reg8 = pci_s_read_config8(dev, 0x48);
-	reg8 |= (1 << 3) | (1 << 4);
-	pci_s_write_config8(dev, 0x48, reg8);
-
-	/* LPC ROM address range 1: */
-	/* Enable LPC ROM range mirroring start at 0x000e(0000). */
-	pci_s_write_config16(dev, 0x68, 0x000e);
-	/* Enable LPC ROM range mirroring end at 0x000f(ffff). */
-	pci_s_write_config16(dev, 0x6a, 0x000f);
-
-	/* LPC ROM address range 2: */
-	/*
-	 * Enable LPC ROM range start at:
-	 * 0xfff8(0000): 512KB
-	 * 0xfff0(0000): 1MB
-	 * 0xffe0(0000): 2MB
-	 * 0xffc0(0000): 4MB
-	 */
-	pci_s_write_config16(dev, 0x6c, 0x10000 - (CONFIG_COREBOOT_ROMSIZE_KB >> 6));
-	/* Enable LPC ROM range end at 0xffff(ffff). */
-	pci_s_write_config16(dev, 0x6e, 0xffff);
-}
-
 void bootblock_early_southbridge_init(void)
 {
 	u32 data;
-
-	hudson_enable_rom();
+	hudson_enable_lpc_rom();
```

Comment:
I'd rather leave that empty line between declarations and statements, even though coreboot code style doesn't explicitly require this. It looks nicer, and may reduce amount of manual conflict resolution during rebase down the line.

Reply:
```suggestion

	hudson_enable_lpc_rom();
```